connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java

```diff
@@ -38,28 +39,36 @@ public SqsQueueConsumer(
   @Override
   public void run() {
     LOGGER.info("Started SQS consumer for queue {}", properties.getQueue().getUrl());
+
     final ReceiveMessageRequest receiveMessageRequest = createReceiveMessageRequest();
     ReceiveMessageResult receiveMessageResult;
     do {
       receiveMessageResult = sqsClient.receiveMessage(receiveMessageRequest);
       List<Message> messages = receiveMessageResult.getMessages();
       for (Message message : messages) {
```

Comment:
Please include the `receiveMessage` in the try catch block. Otherwise the connector stops working when there is a network error for example.